json-endpoints/jetpack/class.jetpack-json-api-get-term-taxonomy-backup-endpoint.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class Jetpack_JSON_API_Get_Term_Taxonomy_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/term-taxonomies/backup      -> $blog_id
```

Comment:
When it comes to term-taxonomy backup
I got the following data base 
<img width="1263" alt="screen shot 2017-03-27 at 22 06 18" src="https://cloud.githubusercontent.com/assets/115071/24389855/bb0bc16a-1339-11e7-9a0d-e0b90fa93205.png">

I think all of the data here is defined in code and not in the database. I am not sure this is intended.